javascript/javascript2/week3/Warmup/location_googlemap/locationMap.js

```diff
@@ -0,0 +1,13 @@
+ // 6. To show that location on a map using fx the Google maps api
+
+ let map;
+
+ function renderLocationOnGoogleMap() {
+   map = new google.maps.Map(document.getElementById('map'), {
+     center: {
+       lat: 55.757,
```

Comment:
instead of hard coding these values, you should be able to do this with     `navigator.geolocation.getCurrentPosition(fn)`